src/stores/modules/User.store.js

```diff
@@ -54,8 +54,7 @@ const actions = {
 
     const {result, success} = await api.user.register(payload.username, payload.password, payload.email);
 
-
-    if(success) {
+    if(result.success) {
```

Comment:
Is there a particular reason why was this changed?  The `success` value returned by `register()` indicates that the HTTP status returned for the API request was 2xx. This is what should be used to check if the API call worked. The `success` field on the result object is deprecated and we are going to remove it at some stage.

See also: https://github.com/TheCacophonyProject/cacophony-api/issues/48

Reply:
yes correct.  I have made that change.